main/src/init/settings.hpp

```diff
@@ -44,11 +44,13 @@ using InitSettings = std::map<std::string, double>;
 //! @brief write @p InitSettings as file attributes of a new file @p path
 inline void writeSettings(const InitSettings& settings, const std::string& path, IFileWriter* writer)
 {
-    if (std::filesystem::exists(path))
-    {
-        throw std::runtime_error("Cannot write settings: file " + path + " already exists\n");
-    }
+    // if (std::filesystem::exists(path))
+    // {
+    //     throw std::runtime_error("Cannot write settings: file " + path + " already exists\n");
+    // }
```

Comment:
This check is there not because it's not possible to continue. It prevents output to an existing file. It's true that H5Part, for example, prevents you from overwriting an existing step, so it _should_ be safe to add additional steps to an existing file. If we're 100% confident that file corruption of existing steps is not possible even in the event of a crash, we can discuss allowing this.

Reply:
Sorry! I didn't realize I've commented these lines (during tests) until you pointed out. Should be fixed.